init.lua

```diff
@@ -20,6 +20,11 @@ maptools.intllib = S
 
 local modpath = minetest.get_modpath("maptools")
 
+maptools.msg = function(player)

```

Comment:
Could be changed to `maptools.drop_msg = function(itemstack, player)`, which allows assigning `on_drop` like this:
`on_drop = maptools.drop_msg`